etcdserver/server.go

```diff
@@ -416,7 +416,7 @@ func NewServer(cfg *ServerConfig) (srv *EtcdServer, err error) {
 		r: raftNode{
 			isIDRemoved: func(id uint64) bool { return cl.IsIDRemoved(types.ID(id)) },
 			Node:        n,
-			ticker:      time.Tick(heartbeat),
+			ticker:      time.NewTicker(heartbeat).C,
```

Comment:
It probably needs a matching Stop() if it's going to be in the spirit of the check.